jolt-core/src/r1cs/snark.rs

```diff
@@ -26,9 +24,10 @@ struct JoltCircuit<F: PrimeField> {
 
 impl<F: PrimeField> Circuit<F> for JoltCircuit<F> {
   fn synthesize<CS: ConstraintSystem<F>>(self, cs: &mut CS) -> Result<(), SynthesisError> {
-    let root = current_dir().unwrap().join("src/r1cs/circuits/jolt/");
-    let r1cs_path = root.join("jolt.r1cs");
-    let wtns_path = root.join("jolt_js/jolt.wasm");
+    let circuit_dir = std::env::var("CIRCUIT_DIR").expect("failed to read CICUIT_DIR, should've been set by build.rs");
+    let circuit_dir = std::path::PathBuf::from(&circuit_dir);
+    let r1cs_path = circuit_dir.join("jolt.r1cs");
+    let wtns_path = circuit_dir.join("jolt_js/jolt.wasm");
 
     let cfg = CircomConfig::new(wtns_path, r1cs_path.clone()).unwrap();
 
```

Comment:
This is probably not relevant to this build PR but we'd need some way to feed in the trace to the `JoltCircuit` struct. I just made it send in the all 0s for now below. How do you imagine we connect to the other parts like `RVTraceRow` later? We probably need to store the trace as a field of `JoltCircuit` cause it needs to accessed within the `synthesize()` function.